### ISSUE TITLE
Example doesn't work with tag v0.1.0, need to tag a new that works

### ISSUE THREAD

**Issue by PallHaraldsson** [IBM/Lale.jl#9]
A)
Works up to:
```
julia> pca     = laleoperator("PCA")
ERROR: UndefVarError: laleoperator not defined
```

The first time around it DID work. And I'm not sure what I did differently. The only thing I can think of, while it shouldn't explain..., is I tried this line again:

```
lalepipe  = (pca + noop) >>  (rfr | treereg )
```

with |> operator to see if it's the same one.

Before I got:

```
julia> lalepipe  = (pca + noop) >>  (rfr | treereg )
PyObject <lale.operators.PlannedPipeline object at 0x7fba2e063d30>

[..]

julia> lalepred  = transform(laletrain,Xreg)
150-element Vector{Float64}:
 0.2615742372881355
```

Should it show PyObject (not really bothering me since I know it's a wrapper), or should it overwrite/hide that? I think the docs should however make it clear, state "a wrapper" rather than "a port" of Python Lale. Or is a full port on the roadmap?

It would be nice to not need pip3 to install in Python (a problem for all Python wrappers I know of), i.e. if it were automatic. A "wrapper" makes expectations more clear, and even clearer if stated on the README page, not just in a link from it.

I'm using Julia 1.6.0.

FYI: I got before in Julia 1.6-DEV:

   Building Lale → `~/.julia/packages/Lale/lM80Z/deps/build.log`
ERROR: SystemError: opening file "/home/pharaldsson_sym/.julia/packages/Lale/lM80Z/deps/build.log": Permission denied

but not when readding, and the new build.log was identical... Shouldn't explain non-DEV Julia problem above.

B)

```
shell> pip3 install lale-gpl
Defaulting to user installation because normal site-packages is not writeable
ERROR: Could not find a version that satisfies the requirement lale-gpl
ERROR: No matching distribution found for lale-gpl
```

Is this something that should work in Julia, or for that matter for  Python?

**Comment by PallHaraldsson** [comment]
@ppalmes 

It's working now (currently) if I do:
```
(@v1.6) pkg> add Lale#main

I see the docs were made to work use the former, that v0.1.0 doesn't support (with main both work, while possibly latter outdated?)::
julia> pca     = laleoperator("PCA")
LalePreprocessor("PCA_ZHD", Dict{Symbol, Any}(:laleobj => PyObject <lale.operators.PlannedIndividualOp object at 0x7f616c8a2ef0>, :name => "PCA_ZHD", :autocomponent => false, :impl_args => Dict{Any, Any}(), :preprocessor => "PCA"))

julia> pca     = LalePreprocessor("PCA")
LalePreprocessor("PCA_UZE", Dict{Symbol, Any}(:laleobj => PyObject <lale.operators.PlannedIndividualOp object at 0x7f616c8a2ef0>, :name => "PCA_UZE", :autocomponent => false, :impl_args => Dict{Any, Any}(), :preprocessor => "PCA"))
```

I probably removed Lale and added back just after you tagged v0.1.0 which has outdated (non-main) docs.

FYI: Also the doc links do not work.

**Comment by ppalmes** [comment]
Hi, thanks for the feedback.

The documentation in README works only for the latest code in Lale#main because there was a 3day grace period for new package to be registered in Julia General registry and we registered the old version which does not yet use the simpler API. Also, the build in travis works for Mac/Linux setup but it seems failing in windows. Until a new version is released, please use `LalePreprocessor` and `LaleLearner` because `laleoperator` will work once we have a new release.

**Comment by ppalmes** [comment]
And yes, the build should not require manual install of lale python. It should be automatic and triggered when you install Lale.jl for the first time. You can try rebuilding to see if it fails: ```using Pkg; Pkg.build("Lale")```

**Comment by ppalmes** [comment]
Indeed, that pipeline part returning a python object is still a work in progress. Ideally, you will directly use `LaleOptimizer(pca >> rf, optimizer arguments...)`. The pipeline expressions are the only ones that return a pyobject which I plan to wrap using macro `@lalepipeline pca >> rf`.

**Comment by ppalmes** [comment]
Oh, you are correct. At the moment, it's just a wrapper of lale at the moment. Thanks for pointing this out. It's now corrected in the README and description.

**Comment by ppalmes** [comment]
Ok, I just submitted a new version to Julia registry which supports the README examples. It should be available soon with tag v0.1.1. Let me know if there are still some issues. Thanks!

**Comment by ppalmes** [comment]
closing this issue. please open it if you still have problem.
